ci-operator/config/openshift/cincinnati-operator/openshift-cincinnati-operator-master.yaml

```diff
@@ -23,6 +27,21 @@ images:
 - dockerfile_path: dev/Dockerfile
   from: base
   to: cincinnati-graph-data-container
+operator:
+  bundles:
+  - as: cincinnati-bundle-container
+    base_index: operator-index
+    context_dir: config
+    dockerfile_path: bundle.Dockerfile
+    update_graph: replaces
+  substitutions:
+  - pullspec: quay.io/openshift/cincinnati-operand:latest
+    with: pipeline:cincinnati_operand_latest
+  - pullspec: quay.io/openshift/cincinnati-operator:latest
+    with: pipeline:cincinnati-operator
+promotion:
+  name: "4.14"
+  namespace: ocp
```

Comment:
I don't think we will need `promotion`

Reply:
Yes, but if we don't promote it, it will be removed once the cluster is removed(around 2 hours). Sometimes, we need to create the operator image by using the cluster-bot, but test it in our local env. So, in case of this image is removed after 2 hours, it's better to promote it.

Reply:
Promotion does not play any role in cluster-bot runs, which is what this PR aims to enable. Promotion setup simply performs post-merge image builds that are promoted to a persistent location.

Bundles (and indices) are not promoteable anyway, so we would only persist actual operator image, which *could* be useful, but we do not definitely want to promote it into `ocp` namespace. Let's omit this from this PR and only enable once we have an actual use to have that image persisted.

Reply:
Thank you Petr and Jian. Dropped.